src/web/lib/decideTheme.ts

```diff
@@ -1,17 +1,28 @@
-import { Design, Pillar, Special } from '@guardian/types';
+import { Pillar, Special } from '@guardian/types';
 
-export const decideTheme = ({
-	pillar,
-	design,
-	isSpecialReport,
-}: {
-	pillar: CAPIPillar;
-	design?: Design;
-	isSpecialReport?: boolean;
-}): Theme => {
-	if (isSpecialReport) return Special.SpecialReport;
-	// We override the pillar to be opinion on Comment news pieces
-	if (design === Design.Comment && pillar === 'news') return Pillar.Opinion;
+export const decideTheme = (format: CAPIFormat): Theme => {
+	const pillar: CAPITheme = format.theme;
+	switch (pillar) {
+		case 'NewsPillar':
+			return Pillar.News;
+		case 'OpinionPillar':
+			return Pillar.Opinion;
+		case 'SportPillar':
+			return Pillar.Sport;
+		case 'CulturePillar':
+			return Pillar.Culture;
+		case 'LifestylePillar':
+			return Pillar.Lifestyle;
+		case 'SpecialReportTheme':
+			return Special.SpecialReport;
+		case 'Labs':
+			return Special.Labs;
+		default:
+			return Pillar.News;
+	}
+};
+
+export const decideNavTheme = (pillar: LegacyPillar): Theme => {
```

Comment:
Ah I don't think this was meant to be exported because I think it's only used within this file. Look s like I copy-pasted something and the export snuck in. I'll go back in and fix that